src/main/kotlin/org/jetbrains/research/ml/coding/assistant/runner/SolutionSpaceRunner.kt

```diff
@@ -0,0 +1,101 @@
+package org.jetbrains.research.ml.coding.assistant.runner
+
+import com.intellij.ide.impl.ProjectUtil
+import com.intellij.openapi.application.ApplicationStarter
+import com.intellij.openapi.components.service
+import com.intellij.openapi.diagnostic.Logger
+import com.xenomachina.argparser.ArgParser
+import org.jetbrains.research.ml.ast.util.getTmpProjectDir
+import org.jetbrains.research.ml.ast.util.sdk.setSdkToProject
+import org.jetbrains.research.ml.coding.assistant.dataset.TaskTrackerDatasetFetcher
+import org.jetbrains.research.ml.coding.assistant.dataset.model.DatasetTask
+import org.jetbrains.research.ml.coding.assistant.report.CodeRepository
+import org.jetbrains.research.ml.coding.assistant.report.OriginalCodeData
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.builder.SolutionSpaceGraphBuilder
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.serialization.SerializationUtils
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.utils.generateImage
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.weightCalculator.CustomEdgeWeightCalculator
+import org.jetbrains.research.ml.coding.assistant.unification.DatasetUnification
+import org.jetbrains.research.ml.coding.assistant.unification.model.DatasetPartialSolution
+import java.io.File
+import java.nio.file.Paths
+import javax.imageio.ImageIO
+import kotlin.system.exitProcess
+
+object SolutionSpaceRunner : ApplicationStarter {
+    private lateinit var inputDir: String
+    private lateinit var outputDir: String
+
+    private val logger = Logger.getInstance(this::class.java)
+
+    override fun getCommandName(): String = "solution-space"
+
+    class TransformationsRunnerArgs(parser: ArgParser) {
+        val input by parser.storing(
+            "-i",
+            "--input_path",
+            help = "Input directory with csv files"
+        )
+
+        val output by parser.storing(
+            "-o",
+            "--output_path",
+            help = "Output directory"
+        )
+    }
+
+    override fun main(args: List<String>) {
+        try {
+            ArgParser(args.drop(1).toTypedArray()).parseInto(SolutionSpaceRunner::TransformationsRunnerArgs).run {
+                inputDir = Paths.get(input).toString().removeSuffix("/")
```

Comment:
I think you can create a function with code: `Paths.get(input).toString().removeSuffix("/")`

Reply:
actually, it is not needed. We use Java's Path and File classes